x-pack/plugins/security_solution/public/common/components/exceptions/add_exception_modal/index.tsx

```diff
@@ -343,6 +356,50 @@ export const AddExceptionModal = memo(function AddExceptionModal({
     return false;
   }, [maybeRule]);
 
+  const OsOptions: Array<EuiComboBoxOptionOption<OsType>> = useMemo((): Array<
+    EuiComboBoxOptionOption<OsType>
+  > => {
+    return [
+      {
+        label: 'Windows',
```

Comment:
I think labels should be `i18n`'d right?

Reply:
yea, will update